tests/release/release.go

```diff
@@ -139,10 +139,10 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 				}
 
 				return release.IsDone() && meta.IsStatusConditionTrue(release.Status.Conditions, "Succeeded")
-			}, avgPipelineCompletionTime, defaultInterval).Should(BeTrue())
+			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("The Release should reference the release PipelineRun.", func() {
+		It("The Release should reference the release PipelineRun.", func(ctx SpecContext) {
```

Comment:
same.

Reply:
Leandro's comment is same as in line 122 

https://github.com/redhat-appstudio/e2e-tests/pull/259#discussion_r1043162107

Reply:
Already resolved all Leandro's comments with him